_installation/interoperability.md

```diff
@@ -0,0 +1,112 @@
+
+Additionally, SQL Server uses different rules than PostgreSQL for resolving an object name without a qualifying schema name. You must take these differences into account when using a PostgreSQL connection to the Babelfish database.
+    
+**Identifiers**
+
+PostgreSQL has a maximum identifier length of 63 characters, while SQL Server supports up to 128. In addition, PostgreSQL has stricter uniqueness requirements for index names. Babelfish handles these restrictions by internally appending or replacing part of such identifiers with a 32-character string representing a hash of the identifier. While this is transparent from the TDS port, the identifier-with-hash is the object name when seen from the PostgreSQL port. 
+    
+For example, an index named `ix1` on table `t1` will internally be named `ix1t1a5111d2a1767bc43a700e9f2162be019` by Babelfish.
+
+**Permissions**
+
+To implement the SQL Server permissions model, Babelfish creates PostgreSQL roles to reflect concepts like SQL Server's database principals. These Babelfish-managed PostgreSQL constructs are transparent from the T-SQL port, but are visible to a PostgreSQL connection. 
+It is important to realize that the Babelfish-managed PostgreSQL roles should not be modified from PostgreSQL, and permissions on Babelfish-created objects should not be granted/revoked from PostgreSQL. While it may currently be technically possible to do so, this could result in a Babelfish instance becoming incompatible with future releases of Babelfish, and could lead to future upgrades failing. It could also lead to an unexpected permissions configuration in current or future versions of Babelfish, as security-related T-SQL statements may not have the expected result due to interference with the actions that were performed in PostgreSQL. 
+
+Babelfish attempts to ensure that a table or other SQL object that is created with T-SQL from a TDS connection in Babelfish, will be accessible with the same permissions when connecting through the PostgreSQL port with the same user, without running any additional PostgreSQL statements (though some of the other considerations listed here, like schema name changes, may still apply). However, the other way around is not guaranteed: when you create an object on the PostgreSQL port, it may or may not be visible or accessible from a TDS connection, or it may be accessible with different permissions. Therefore, if you plan to access schemas and objects from the PostgreSQL port, you should create those objects in T-SQL on the TDS port. 
```

Comment:
attempts to ensure ==> ensures. It we go with 'attempts' we must point out when it doesn't and currently no such cases apply